app/uk/gov/hmrc/emcstfe/models/response/ErrorResponse.scala

```diff
@@ -102,4 +102,5 @@ object ErrorResponse {
     val message = s"no action found in the request"
   }
 
+  case class PreValidationError(message: String) extends ErrorResponse
```

Comment:
I don't think this is being used?

Reply:
Removed https://github.com/hmrc/emcs-tfe/pull/139/commits/da84e485b1ba035d4f2711bd9fd2268a6e4935be